Reference/materials_expanded.json

```diff
@@ -0,0 +1,4178 @@
+{
```

Comment:
We do..? I am not sure, but I plan to add all of these as guide items, so I can check it soon.

Reply:
I just realized that maybe your intention was more to keep track of the Wynncraft api as just a Reference service, not that we should use it in Wynntils? If so, then it sounds great. That'll mean that if they ever change the values, we will notice.

Reply:
We might as well just use this data and not generate it ourselves, might be just easier and more obvious in the long run..

Reply:
I had to go and check. :) Yes we do, see `MaterialProfile`.

Reply:
I compared the information there with this file. The difference is that here they list "material", that is, the minecraft item id, and "craftable", what kind of items you can make from it.

There's a lot of redundancy in this file. I think it would be sad to lose our compact representation. If we do start incorporating this file, I'd vote for that we only read what we miss (the minecraft item id+damage). And if we want to add information that paper can be used to craft scrolls, helmets and chestplates, we should add so on an abstract level, saying this about PAPER in general, not duplicating it across all individual paper tiers and source tree types.